images/echo/cloudbuild.yaml

```diff
@@ -1,17 +0,0 @@
-timeout: 1200s
-options:
```

Comment:
i thought we were keeping the cloudbuilds and adding the GHA functionality?

Reply:
I have removed just from this image, as it is a bit borked and the name from it is different from folder, etc etc